packages/muelu/src/Utils/MueLu_PerfUtils_decl.hpp

```diff
@@ -54,7 +54,7 @@
 #include <Xpetra_Import_fwd.hpp>
 #include <Xpetra_Matrix_fwd.hpp>
 
-//#include "MueLu_Utilities_fwd.hpp"
+#include "MueLu_PerfUtils_fwd.hpp"
 
```

Comment:
Why do we need the forward declaration here in the decl file of the same class? Is it because of the Hierarchy::Manager? Couldn't we include it there before the MueLu_PerfUtils.hpp include?